web/explorer/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
-        <link rel="icon" href="/public/favicon.ico" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
does vite rewrite these or is this also an absolute path that we should update?

Reply:
No, this is already the correct absolute path and does not need to be updated. [Context](https://stackoverflow.com/a/71400083).